src/app/pages/dashboard/components/widget-backup/widget-backup.component.html

```diff
@@ -0,0 +1,185 @@
+<div class="widget stats-widget" ngClass.xs="double-height xs">
+  <div class="card-container front">
+    <mat-card
+      class="front"
+      fxLayout="row"
+      fxLayoutWrap
+      fxLayoutAlign="space-between stretch"
+    >
+      <mat-card-content class="card">
+        <ix-drag-handle [showReorderHandle]="showReorderHandle">
+          <ng-content [select]="[cdkDragHandle]"></ng-content>
+
+          <mat-toolbar-row>
+            <!-- Mobile Only: Exit widget -->
+            <button
+              *ngIf="screenType === ScreenType.Mobile"
+              class="mat-card-title-text"
+              mat-icon-button
+              color="default"
+              ixTest="help-back"
+              (click)="goBack()"
+            >
+              <ix-icon name="chevron_left" class="back-arrow"></ix-icon>
+            </button>
+
+            <h3 class="mat-card-title-text">
+              <ix-drag-handle [showReorderHandle]="showReorderHandle">
+                <div class="card-title-text">
+                  <span *ngIf="showReorderHandle" class="grip"></span>
+                  <span>{{ 'Backup Tasks' | translate }}</span>
+                </div>
+              </ix-drag-handle>
+            </h3>
+
+            <div *ngIf="!!allCount" class="status-container">
+              <ix-icon
+                *ngIf="!failedCount"
+                name="mdi-check-circle"
+                class="icon healthy"
+              ></ix-icon>
+              <ix-icon
+                *ngIf="!!failedCount"
+                name="mdi-alert"
+                class="icon warning"
+              ></ix-icon>
+              <span *ngIf="!!failedCount">
+                {{ '{failed} of {all} tasks failed' | translate : { failed: failedCount, all: allCount } }}
```

Comment:
I think variable names are better self-explanatory than placeholders.
Let's update placeholder names to match variables.

Also, `{all} tasks` need plural support.

Reply:
Plural support seems required for each occurrence of `tasks` in this PR. Please take a look.